app/src/main/res/layout/fragment_new_message.xml

```diff
@@ -476,22 +476,26 @@
             app:layout_constraintTop_toBottomOf="@+id/compositionNestedScrollView"
             tools:visibility="visible">
 
-            <TextView
-                style="@style/BodySmall"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_marginHorizontal="@dimen/marginStandardMedium"
-                android:layout_weight="1"
-                android:text="@string/externalDialogTitleRecipient"
-                android:textColor="@color/externalTagOnBackground" />
-
             <com.google.android.material.button.MaterialButton
                 android:id="@+id/informationButton"
-                style="@style/IconButtonSmall"
+                style="@style/TextButton"
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
                 android:layout_marginEnd="@dimen/marginStandardMedium"
+                android:fontFamily="@font/suisseintl_medium"
+                android:minHeight="0dp"
+                android:paddingVertical="0dp"
+                android:text="@string/externalDialogTitleRecipient"
+                android:textColor="@color/primaryTextColor"
                 app:icon="@drawable/ic_external_information"
+                app:iconGravity="end"
                 app:iconTint="@color/externalTagOnBackground" />
 
+            <Space
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_weight="1" />
```

Comment:
A `ConstraintLayout` would be better instead of a Space IMO.

Reply:
A ConstraintLayout is overkill when items are simply placed side by side like this I think. It requires an unnecessary amount of constraint to achieve this very simple result. Directly using a LinearLayout also helps you understand how the views are gonna be arranged where as using a ConstraintLayout requires you to go and apply every constraint in your mind to understand where views are placed from one another. In this sens, it's more meaningful/easy to grasp I think